qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/AbstractRollingUpgradeTestCase.java

```diff
@@ -71,7 +71,7 @@ public static ClusterType instance(String value) {
     }
 
     protected final ClusterType getClusterType() {
-        return ClusterType.instance(System.getProperty(BWCSUITE_CLUSTER));
+        return ClusterType.instance(System.getProperty(BWCSUITE_CLUSTER, "invalid_cluster_type"));
```

Comment:
What does this do?

Reply:
If this property, `BWCSUITE_CLUSTER` is not set then it will return null and the ClusterType switch case can't handle null. So, I thought instead we can return `invalid_cluster_type` as default value for that property which will throw an exception later in the switch case as it is not a valid cluster type.

Instead do you want me to check for null after fetching that property and throw an exception here?

Reply:
I think throwing an exception is better